src/components/Grid/Grid.vue

```diff
@@ -422,37 +422,34 @@ export default {
               })));
```

Comment:
No actual changes to this file. Just got rid of the comma separated variable declarations to help with debugging.